### PR TITLE
Added instructions for increasing the priority of tickets

### PR DESCRIPTION
**Pull request by icelander**
Signed-off-by: Paul Rothrock <paul@movetoiceland.com>

### CODE REVIEW

File: source/process/support.md [hunk 1]
```diff
@@ -193,3 +193,17 @@ File locations:
...
 - The default data directory is `/var/opt/gitlab/mattermost/data`.
 - Other Mattermost resources (i18n, email templates, webapp code, etc) are located in `/opt/gitlab/embedded/service/mattermost`.
+
+## Escalating Support Requests (E10/E20)
+
+Enterprise customers have access to support SLAs based on ticket priority. To indicate your ticket's urgency, respond with the business impact level in this format: `SLA Level: #` where `#` is from 1 to 4 based on the following criteria:
+
+**SLA Level: 1 - Critical Business Impact** – Critical issue on production system preventing business operations. A large number of users are prevented from working, and no procedural workaround is available. System hangs or crashes, critical functionality not available, data loss or data corruption, large number of end users blocked from work. Impact escalating quickly.
+
+**SLA Level: 2 - Major Business Impact –** Major issue on production system severely impacting business operationSignificant performance degradation, important functionality not available, small number of users blocked from work, impact is escalating
```
Comment (amyblais) [inline source/process/support.md]:
```suggestion
**SLA Level: 2 - Major Business Impact –** Major issue on production system severely impacting business operation. Significant performance degradation, important functionality not available, small number of users blocked from work, impact is escalating.
```

File: source/process/support.md [hunk 2]
```diff
@@ -193,3 +193,17 @@ File locations:
...
+
+**SLA Level: 3 - Moderate Business Impact -** Moderate issue causing a partial or non-critical loss of functionality on production system. A small number of users are affectedSome system functions not available, minor performance degradation, small number of users impacted, impact is not escalating
```
Comment (amyblais) [inline source/process/support.md]:
```suggestion
**SLA Level: 3 - Moderate Business Impact -** Moderate issue causing a partial or non-critical loss of functionality on production system. A small number of users are affected. Some system functions not available, minor performance degradation, small number of users impacted, impact is not escalating.
```

File: source/process/support.md [hunk 3]
```diff
@@ -193,3 +193,17 @@ File locations:
...
+
+**SLA Level: 4 - Minor Business Impact -** Minor issue on non-production system or question, comment, feature request, documentation issue or other non-impacting issue. Incorrect product behavior without impact, or product question or enhancement
```
Comment (amyblais) [inline source/process/support.md]:
```suggestion
**SLA Level: 4 - Minor Business Impact -** Minor issue on non-production system or question, comment, feature request, documentation issue or other non-impacting issue. Incorrect product behavior without impact, or product question or enhancement.
```

**Comment by hanzei** [comment]
@icelander Gentle reminder about this PR

**Comment by hanzei** [comment]
@icelander I'm closing this PR for now, but keeping the branch. Please just reopen it, if you still want to work on this.

**Pull request closed without merging**